test/functional/wallet_signer.py

```diff
@@ -111,8 +111,10 @@ def test_valid_signer(self):
         assert_equal(address_info['hdkeypath'], "m/44'/1'/0'/0/0")
 
         self.log.info('Test walletdisplayaddress')
-        result = hww.walletdisplayaddress(address1)
-        assert_equal(result, {"address": address1})
+        for address in [address1, address2, address3]:
+            print(address)
```

Comment:
c83ab3059 The printed addresses correspond to the updated mocks. I think you intended to remove this line, though.